cookbook/mass_edition/register_a_new_mass_edit_action.rst

```diff
@@ -153,6 +153,16 @@ The Job has to be in your database, so add it to your fixtures:
 
     To better understand how to handle this, you can read this chapter: :ref:`add-your-own-data`
 
+If your installation is already set up, use the `akeneo:batch:create-job` command:
+
+.. code-block:: bash
+    php app/console akeneo:batch:create-job "Akeneo Mass Edit Connector" "mass_edit_capitalize_values" "mass_edit" "capitalize_values" '{}' "Mass capitalize products value"
+
+.. warning::
+
+    If you are using Akeneo PIM EE, don't forget to add permissions in `pimee_security_job_profile_access` to your job profile.
```

Comment:
I think you should add how to do this:
- updates fixtures and new install
- insert statement in MySQL to update this table

Otherwise we are kind of frustrated, I'm in EE and I don't know how/where to do this.